bokehjs/src/coffee/models/tools/gestures/wheel_zoom_tool.coffee

```diff
@@ -19,20 +19,20 @@ class WheelZoomToolView extends GestureTool.View
     @_scroll(e)
 
   _scroll: (e) ->
-    frame = @plot_model.get('frame')
+    frame = @plot_model.frame
     hr = frame.get('h_range')
     vr = frame.get('v_range')
```

Comment:
@bokeh/core, I removed quite a few uses of `get()` and `mget()` as @bryevdv [requested in an earlier comment] (https://github.com/bokeh/bokeh/pull/4841#discussion_r71641321).  I was surprised that `CartesianFrame` wouldn't let me access `h_range` and `v_range` without `get()`.

Is this because `@define_computed_property()` and `@define()` properties are different somehow?  Or, am I missing something else?